kvdb-web/Cargo.toml

```
@@ -0,0 +1,42 @@
+[package]
+name = "kvdb-web"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+repository = "https://github.com/paritytech/parity-common"
+description = "A key-value database for use in browsers"
+documentation = "https://docs.rs/kvdb-web/"
+license = "GPL-3.0"
+edition = "2018"
+
+[dependencies]
+wasm-bindgen = "0.2.49"
+js-sys = "0.3.26"
+kvdb = { version = "0.1", path = "../kvdb" }
+kvdb-memorydb = { version = "0.1", path = "../kvdb-memorydb" }
+futures-preview = "0.3.0-alpha.18"
```

Could you change this to alpha.17? We are forcing version alpha.17 in Substrate because of a compilation error with alpha.18.

```suggestion
futures-preview = "0.3.0-alpha.17"
```